vmdb/app/models/host.rb

```diff
@@ -96,7 +96,7 @@ class Host < ActiveRecord::Base
   serialize                 :settings
 
   def settings
-    super || (settings = VMDB::Config.new("hostdefaults").get(:host))
+    super || VMDB::Config.new("hostdefaults").get(:host)
```

Comment:
This might be a bug...I think the code was supposed to be `super || (self.settings = VMDB::Config.new("hostdefaults").get(:host))`